frontend/src/components/menu/LogoImage.test.ts

```diff
@@ -32,10 +38,8 @@ describe('LogoImage', () => {
     })
 
     describe('size large', () => {
-      beforeEach(async () => {
-        await wrapper.setProps({
-          size: 'large',
-        })
+      beforeEach(() => {
+        wrapper = Wrapper({ size: 'large' })
```

Comment:
Is there a missing

```suggestion
        wrapper = await Wrapper({ size: 'large' })
```

maybe?

Reply:
Don't think so, mount is not async.